dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -383,6 +384,28 @@ public Contentlet find(final String inode, final User user, final boolean respec
         }
     }
 
+    @CloseDBIfOpened
+    @Override
+    public Contentlet find(final String inode, final User user, final String variantName, final boolean respectFrontendRoles)
+        throws DotDataException, DotSecurityException {
+        final Contentlet contentlet = contentFactory.find(inode, variantName);
+        if (contentlet == null) {
+            return null;
```

Comment:
Shouldn't we use Optionals instead of returning null?

Reply:
I removed this method 

https://github.com/dotCMS/core/pull/24182#discussion_r1117096986